pkgs/development/python-modules/pyrevolve/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, contexttimer
+, versioneer
+, cython
+, numpy
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "pyrevolve";
+  version = "2.2";
+
+  src = fetchFromGitHub {
+    owner = "devitocodes";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-5a4zvyf2vfz8aI6vFMI2vxekYrcUi/YuPFvZnUOx+Zs=";
+  };
+
+  nativeBuildInputs = [ versioneer cython ];
+  propagatedBuildInputs = [ contexttimer ];
+
+  checkInputs = [ pytest numpy ];
+  checkPhase = ''
+    pytest
+  '';
```

Comment:
```suggestion
  checkInputs = [ pytestCheckHook numpy ];
```

Reply:
The tests are failing with the hook e.e

Reply:
Maybe nice to note that above the checkPhase to make it clearer why it's done like this.